star-ros2/src/star_safety_monitor/test/test_safety_monitor.cpp

```
@@ -189,26 +181,22 @@ TEST_F(SafetyMonitorTest, OdometrySubscription)
   EXPECT_TRUE(true);
 }
 
-TEST_F(SafetyMonitorTest, BatterySafetyChecks)
-{
+TEST_F(SafetyMonitorTest, BatterySafetyChecks) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)